README.rst

```diff
@@ -0,0 +1,217 @@
+This is a userscript intended to be used with greasemonkey or tampermonkey.
+
+It enriches the github pull request page with so called 'incremental diffs'.
+This means that you will get a diff for changes pushed to that pull request,
+showing the difference before and after the push.
+
+This is particular useful for situations where the reviewee was asked to do some
+changes in their existing commits (not additional commits). The reviewer can
+then use this to see those changes directly, not having to go through all the
+commits again, making sure the requested changes were addressed.
+
+All this works by using github webhook, an external server and this script.
+
+User Setup
+==========
+
+A working installation of greasemoneky or tampermonkey is required.
+
+To install the script, just follow this `link <https://raw.githubusercontent.com/sociomantic-tsunami/kelpie/master/incremental-pr-diffs.user.js>`_.
+
+After navigating to any pull request page, the script will ask you to setup a
+personalized access token for github so it can access the github API.
+
+The hash data url is optional and not required for the normal user. If you do
+need it, your repository admin should have told you about it.
+
+You can change your configuration settings later anytime by double-clicking on
+the ``Incremental Diffs Active`` headline at the bottom of the right sidebar.
+
+The rest of this README talks about how to set things up for the repository
+admin and can savely be ignored by the user.
+
+Bootstrapping
+=============
+
+Overview
+~~~~~~~~
+
+You will need to setup a server that will receive github webhook calls and that
+will provide the recorded hashes for each push to a pull request.
+
+The solution outlined here is just one possible out of many.
```

Comment:
I would summarize here what is the stack used by this particular setup, something like "for this example setup we use an nginx server to proxy the requests to the webhooks server and Uwsgi as an interface between both. But actually, I think this might be too complex as a setup example, I would just suggest to use the bare webhooks server without an nginx on top. This README is only supposed to offer a very simplistic setup to get you started, not a complex deployment stack.